openpype/scripts/ocio_wrapper.py

```diff
@@ -106,11 +106,36 @@ def _get_colorspace_data(config_path):
 
     config = ocio.Config().CreateFromFile(str(config_path))
 
-    return {
-        c_.getName(): c_.getFamily()
-        for c_ in config.getColorSpaces()
+    colorspace_data = {
+        "colorspace": {
+            color.getName(): {
+                "family": color.getFamily(),
+                "categories": list(color.getCategories()),
+                "aliases": list(color.getAliases()),
+                "equalitygroup": color.getEqualityGroup(),
+            }
+            for color in config.getColorSpaces()
+        }
     }
 
+    # add looks
+    looks = config.getLooks()
+    if looks:
+        colorspace_data["look"] = {
+            look.getName(): {"process_space": look.getProcessSpace()}
+            for look in looks
+        }
+
+    # add roles
+    roles = config.getRoles()
+    if roles:
+        colorspace_data["role"] = {
+            role: {"colorspace": colorspace}
+            for (role, colorspace) in roles
+        }
```

Comment:
```suggestion
    looks = config.getLooks()
    if looks:
        colorspace_data["looks"] = {
            look.getName(): {"process_space": look.getProcessSpace()}
            for look in looks
        }

    # add roles
    roles = config.getRoles()
    if roles:
        colorspace_data["roles"] = {
            role: {"colorspace": colorspace}
            for (role, colorspace) in roles
        }
```
Seems weird that these keys are singular even though they contain multiple entries?

Reply:
Also, what's holding us from **always** adding the `looks` and `roles` key in the returned data - even when e.g. empty? At least then our ouput dict structure is more consistent.